test/unit/test_sap_adt_behaviordefinition.py

```
@@ -99,5 +101,158 @@ def test_adt_bdef_activate(self):
+        bdef = sap.adt.BehaviorDefinition('CONNECTION', name='ZMYBDEF')
+        template = ADTTemplate([
+            ADTTemplateProperty('base_bdef', 'R_PRODUCTTP'),
+            ADTTemplateProperty('interface_bdef'),
+        ])
+        bdef.template = template
+        self.assertEqual(bdef.template, template)
+
+    def test_bdef_create_without_template(self):
+        conn = Connection([Response(text='', status_code=201)])
+        metadata = sap.adt.ADTCoreData(language='EN', master_language='EN',
+                                        package='MYPACKAGE', responsible='DEVELOPER')
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)